notebooks/en/trl_grpo_reasoning_advanced_reward.ipynb

```diff
@@ -0,0 +1,1004 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        num_generations=4,  # Reduce to save memory
</pre><p>reducing num_generation doesn't save memory (it used to be the case, but now, generation and training batches are decoupled). I think you can safely keep the default value (8)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>